localMatchApp/sources/mainwindow.cpp

```diff
@@ -0,0 +1,55 @@
+#include "../headers/mainwindow.h"
+#include "ui_mainwindow.h"
+#include "../headers/signup.h"
+#include "../headers/homewindow.h"
+#include "QDesktopWidget"
+#include <QMessageBox>
+
+MainWindow::MainWindow(QWidget *i_parent) :
+    QMainWindow(i_parent),
+    ui(new Ui::MainWindow)
+{
+    int fixedWidthMainWindow = 500;
+    int fixedHeightMainWindow = 450;
+    ui->setupUi(this);
+    QPixmap pix (":/images/logo.svg");
+    ui->pictureLogo->setPixmap(pix);
+    this->setFixedWidth(fixedWidthMainWindow);
+    this->setFixedHeight(fixedHeightMainWindow);
+    this->setGeometry(
+        QStyle::alignedRect(
+            Qt::LeftToRight,
+            Qt::AlignCenter,
+            this->size(),
+            qApp->desktop()->availableGeometry()
+        )
+    );
+    connect(ui->buttonSignUp,SIGNAL(clicked()), this, SLOT(onButtonSignUpClicked()));
+    connect(ui->buttonLogin,SIGNAL(clicked()), this, SLOT(onButtonLoginClicked()));
+}
+
+MainWindow::~MainWindow()
+{
+    delete ui;
+}
+
+void MainWindow::onButtonSignUpClicked()
+{
+    SignUp signUpWindow;
+    signUpWindow.setModal(true);
+    signUpWindow.exec();
+}
+
+void MainWindow::onButtonLoginClicked()
+{
+    if(ui->lineEditUser->text()=="masolano" && ui->lineEditPassword->text()=="123"){
```

Comment:
Add a comment for these hardcoded values.
Add a TODO comment indicating this must be changed.